contracts/0.8.25/vaults/VaultHub.sol

```diff
@@ -334,6 +354,32 @@ abstract contract VaultHub is PausableUntilWithRoles {
         emit VaultRebalanced(msg.sender, sharesToBurn);
     }
 
+    /// @notice Forces validator exit from the beacon chain when vault health ratio is below 100%
+    /// @param _vault The address of the vault to exit validators from
+    /// @param _pubkeys The public keys of the validators to exit
+    /// @param _refundRecepient The address that will receive the refund for transaction costs
+    /// @dev    When a vault's health ratio drops below 100%, anyone can force its validators to exit the beacon chain
+    ///         This returns the vault's deposited ETH back to vault's balance and allows to rebalance the vault
+    function forceValidatorExit(
+        address _vault,
+        bytes calldata _pubkeys,
+        address _refundRecepient
+    ) external payable {
+        if (msg.value == 0) revert ZeroArgument("msg.value");
+        if (_vault == address(0)) revert ZeroArgument("_vault");
+        if (_pubkeys.length == 0) revert ZeroArgument("_pubkeys");
+        if (_refundRecepient == address(0)) revert ZeroArgument("_refundRecepient");
+        if (_pubkeys.length % PUBLIC_KEY_LENGTH != 0) revert InvalidPubkeysLength();
+        _requireUnhealthy(_vault);
+
+        uint256 numValidators = _pubkeys.length / PUBLIC_KEY_LENGTH;
+        uint64[] memory amounts = new uint64[](numValidators);
+
+        IStakingVault(_vault).triggerValidatorWithdrawal{value: msg.value}(_pubkeys, amounts, _refundRecepient);
+
+        emit ForceValidatorExitTriggered(_vault, _pubkeys, _refundRecepient);
+    }
+
     function _disconnect(address _vault) internal {
```

Comment:
:warning: backlog: disconnect immediately (once asynchronous oracle is here)?

@vp4242 @tamtamchik